docs/examples/django/pages/apps.py

```diff
@@ -0,0 +1,18 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from django.apps import AppConfig
```

Comment:
This isn't used, for it to be used it needs to be referenced in the init file. But I think it can be omitted entirely, as it doesn't do much here

Reply:
Running the example without this file gave me this error: `ModuleNotFoundError: No module named 'pages.apps'`

Reply:
Then I would add it linking to this in the init file (according to [the django docs](https://docs.djangoproject.com/en/3.0/ref/applications/#for-application-authors))

Reply:
Added the link, please take a look